snuba/datasets/replays.py

```diff
@@ -0,0 +1,7 @@
+from snuba.datasets.dataset import Dataset
+from snuba.datasets.entities import EntityKey
+
+
+class ReplaysDataset(Dataset):
+    def __init__(self) -> None:
+        super().__init__(default_entity=EntityKey.REPLAYS)
```

Comment:
https://github.com/getsentry/snuba/blob/672406e88f6f1199cad7ad43b43d0fa9996078ce/snuba/datasets/dataset.py#L46-L54

Please override `is_experimental`. This will allow you to deploy a broken version of the dataset without impacting snuba production deployment